pkg/pipeline/transform/transform_network.go

```diff
@@ -81,6 +83,17 @@ func (n *Network) Transform(inputEntry config.GenericMap) (config.GenericMap, bo
 				}
 				outputEntry[rule.Output+"_Evaluate"] = true
 			}
+		case api.OpAddLabelIf:
```

Comment:
@KalmanMeth this is an "expensive" operation and I envision that we will also want to add "static" labels so can you add another operation that is not with "if" but just a simple AddLabel operation?

Reply:
done